earth2mip/_channel_means.py

```diff
@@ -0,0 +1,237 @@
+channel_means = {
```

Comment:
I dont like these being in the root of the module, but there is not a better spot at the moment. I'll probably work on getting these perturbation methods more modular this week.